packages/models/src/lib/table.ts

```diff
@@ -25,7 +26,6 @@ export const tableSchema = (description = 'Table information') =>
   z.object(
     {
       headings: z.array(tableHeadingSchema).optional(),
-      alignment: z.array(tableAlignmentSchema).optional(),
       rows: z.array(tableRowSchema),
     },
```

Comment:
I'd suggest renaming `headings` property to `columns`. It now supports specifying alignment, so it doesn't only affect how the heading is rendered. Also, I think it's more elegant to have `rows` and `columns` naming, as these terms are commonly used together.

Reply:
renamed